src/main/java/it/gov/pagopa/payhub/auth/utils/JWTValidator.java

```diff
@@ -40,7 +54,7 @@ public Map<String, String> validate(String token, String urlJwkProvider) {
         } catch (com.auth0.jwt.exceptions.TokenExpiredException e){
             throw new TokenExpiredException(e.getMessage());
         } catch (JwkException | JWTVerificationException ex) {
-            throw new InvalidTokenException(AuthConstants.ExceptionCode.INVALID_TOKEN, "The token is not valid", true, ex);
+            throw new InvalidTokenException(AuthErrorDTO.CodeEnum.INVALID_TOKEN.getValue(), "The token is not valid", true, ex);
```

Comment:
the first param is an openapi implementation detail that should not be visible in the service layer

Reply:
The code we are using here is not in any way part of HTTPstatus,  is just a business code, not a presentation layer. If we put a string we can risk to put some inexistent Enum

Reply:
sorry, let me rephrase: do we need to put that string/enum there? Isn't that code already codified in the exception itself? 
e.g. `InvalidTokenException` codifies a `INVALID_TOKEN` code. Why we explicitly put that code in the throw?